src/main/java/org/magicdgs/readtools/RTHelpConstants.java

```diff
@@ -92,19 +84,20 @@ private RTHelpConstants() {}
     private static Map<String, String> getSuperCategoryMap() {
         if (groupToSuperCategory == null) {
             // TODO: initialize with GATK's and/or Picard's supercat map
+            // TODO: https://github.com/magicDGS/ReadTools/issues/370
```

Comment:
Should decide if this is still applicable

Reply:
This won't happen: removed the comment and edit the PR to reflect that it is closed.